CONTRIBUTING.md

```diff
@@ -55,6 +55,15 @@ $ git add package.json
 $ npm run commit
 ```
 
+### Breaking Changes
+
+`npm run commit` will ask you if a change is a breaking change. This is a non-exhaustive list of what we consider breaking changes for `helix-pipeline`
+
+* any change that will remove or rename HTML attributes in the generated HTML output
+* any change that will remove or rename HTML tags in the generated HTML output
+* any change that will remove or rename configuration settings in `secrets`
+* any change that will remove or rename properties exported through `index.js module.exports`
+
```

Comment:
We might also add:
* any change that will alter the structure of the generated HTML output

Reply:
I think the first two lines cover that more succinctly. It's not that any change is a breaking change, only changes that remove or rename tags or attributes. Your selectors should still work for additive changes to the HTML.

Reply:
I was thinking about something like
```html
<div class="main">
  <p>Foo</p>
</div>
```
with the following customer CSS:
```css
div.main > p {
  font-size: 1.5em;
}
```
We might change the structure of the HTML by adding tags:
```html
<div class="main">
  <div>
    <p>Foo</p>
  </div>
</div>
```
This will break the DOM contract.